### PR TITLE
Rationalise CSS comments

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done
- Remove redundant SASS docs comments
- Replace all `//` comments with `///`

## QA

Check I've not deleted anything other than the above.

### CODE REVIEW

File: scss/_base_typography.scss [hunk 1]
```diff
@@ -1,11 +1,5 @@
-////
-/// @author       Web Team at Canonical Ltd
-/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-typography
-/// @since        0.0.3
-////
-
-/// Typographic element styles
-/// @group Typography
+// Typographic element styles
+// @group Typography
```
Comment (anthonydillon) [inline scss/_base_typography.scss]: Did you mean to leave this group comment?

File: scss/_base_typography.scss [hunk 2]
```diff
@@ -71,8 +65,8 @@
     font-weight: 300;
   }
 
-  /// Links
-  /// @section links
+  // Links
+  // @section links
```
Comment (anthonydillon) [inline scss/_base_typography.scss]: You removed other section comments. Should this remain

File: scss/_base_typography.scss [hunk 3]
```diff
@@ -191,8 +185,8 @@
     font-size: .813em;
   }
 
-  /// Base typography
-  /// @section type
+  // Base typography
+  // @section type
```
Comment (anthonydillon) [inline scss/_base_typography.scss]: Ditto

File: scss/_overrides_helpers.scss [hunk 4]
```diff
@@ -1,11 +1,5 @@
-////
-/// @author       Web Team at Canonical Ltd
-/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-helpers
-/// @since        0.0.52
-////
-
-/// A collect of functions to aid and enhance the users abililty
-/// @group Helpers
+// A collect of functions to aid and enhance the users abililty
+// @group Helpers
```
Comment (anthonydillon) [inline scss/_overrides_helpers.scss]: Group again?

File: scss/_patterns_boxes.scss [hunk 5]
```diff
@@ -1,16 +1,3 @@
...
-////
-
-/// Box class
-/// @group Boxes
-/// @example
-///   <div class="box">
-///     <h2>A heading</h2>
-///     <p>Your content here</p>
-///   </div>
 @mixin vf-boxes() {
```
Comment (anthonydillon) [inline scss/_patterns_boxes.scss]: Removed all the comments here?

File: scss/_patterns_code.scss [hunk 6]
```diff
@@ -31,14 +20,14 @@
...
-///     <code class="code-block__line">...</code>
-///   </pre>
+// Code block styles
+// @group Code
+// @example
+//   <pre class="code-block">
+//     <code class="code-block__line">...</code>
+//     <code class="code-block__line">...</code>
+//     <code class="code-block__line">...</code>
+//   </pre>
```
Comment (anthonydillon) [inline scss/_patterns_code.scss]: Left an example and group here

File: scss/_patterns_code.scss [hunk 7]
```diff
@@ -76,13 +65,13 @@
...
-///   <input class="code-snippet__input" value="sudo apt-get install" readonly="readonly">
-///   <button class="code-snippet__copy-button">Copy to clipboard</button>
-/// </div>
+// Code snippet styles
+// @group Code
+// @example
+// <div class="code-snippet">
+//   <input class="code-snippet__input" value="sudo apt-get install" readonly="readonly">
+//   <button class="code-snippet__copy-button">Copy to clipboard</button>
+// </div>
```
Comment (anthonydillon) [inline scss/_patterns_code.scss]: Example and group here

File: scss/_patterns_forms.scss [hunk 8]
```diff
@@ -128,8 +121,8 @@
   }
 }
 
-/// Default form input element styles
-/// @group Forms
+// Default form input element styles
+// @group Forms
```
Comment (anthonydillon) [inline scss/_patterns_forms.scss]: Remaining group

File: scss/_patterns_forms.scss [hunk 9]
```diff
@@ -157,8 +150,8 @@
   }
 }
 
-/// Default form checkbox and radio styles
-/// @group Forms
+// Default form checkbox and radio styles
+// @group Forms
```
Comment (anthonydillon) [inline scss/_patterns_forms.scss]: Remaining group

File: scss/_patterns_forms.scss [hunk 10]
```diff
@@ -176,8 +169,8 @@
   }
 }
 
-/// Disabled form elements
-/// @group Forms
+// Disabled form elements
+// @group Forms
```
Comment (anthonydillon) [inline scss/_patterns_forms.scss]: Group

File: scss/_settings_global.scss [hunk 11]
```diff
@@ -1,120 +1,120 @@
-/// standard colors
+// standard colors
```
Comment (anthonydillon) [inline scss/_settings_global.scss]: These comment could be deleted

File: scss/_settings_global.scss [hunk 12]
```diff
@@ -1,120 +1,120 @@
...
 
-/// Heading font family
-/// default: $base-font-family
+// Heading font family
+// default: $base-font-family
 $heading-font-family:   $base-font-family !default;
 
-/// transparent to use throughout the site
+// transparent to use throughout the site
 $transparent:           transparent !default;
```
Comment (anthonydillon) [inline scss/_settings_global.scss]: Duplicated variables. You moved them to a colours settings file. These should be removed

**Comment by anthonydillon** [comment]
Thanks for this @barrymcgee, few comments on code.

File: scss/_settings_global.scss (hunk 11 above)
Comment (barrymcgee) [inline scss/_settings_global.scss]: This whole file will be split up and deleted in the next PR.

File: scss/_settings_global.scss (hunk 12 above)
Comment (barrymcgee) [inline scss/_settings_global.scss]: This colors settings file shouldn't be in this PR - good spot.

**Pull request merged**